service/src/main/resources/static/openapi.yml

```diff
@@ -0,0 +1,51 @@
+openapi: 3.0.3
+info:
+  title: Terra Batch Analysis
+  description: Manage batch analyses, workflows, and submissions in userspace
+  version: 0.0.1
+paths:
```

Comment:
I'm sure you already know this, but in order to deploy in K8S you need at least `/status` as well.